TestCases/rans/actuatordisk_bem/actuatordisk_bem.cfg

```
@@ -0,0 +1,256 @@
+%(inlet face marker, outlet face marker, X_CG of inlet, Y_CG of inlet, Z_CG of inlet ,
+% X_CG of outlet, Y_CG of outlet, Z_CG of outlet), X_CG, Y_CG, Z_CG of both inlet and
+% outlet markers should be same
+MARKER_ACTDISK_BEM_CG= ( ACTDISK_IN, ACTDISK_OUT, 0.0, 0.0, 0.0,  0.0, 0.0, 0.0 )
+%Actuator Disk Axis
+%(inlet face marker, outlet face marker,
+% axis_x, axis_y, axis_z of inlet, axis_x, axis_y, axis_z of outlet),
+% axis of both inlet and outlet markers should be same
+MARKER_ACTDISK_BEM_AXIS= ( ACTDISK_IN, ACTDISK_OUT, 1.0, 0.0, 0.0,  1.0, 0.0, 0.0 )
+% Input file containing the propeller section data and aerodynamic details of blade sections
+BEM_PROP_FILENAME = prop_geom_alfclcd_data.txt
+% Propeller blade angle (Degrees) at 0.75*Radius for the current calculation
+BEM_PROP_BLADE_ANGLE = 23.9
+% BEM calculation frequency
+BEM_FREQ = 40
+%
+% Far-field boundary marker(s) (NONE = no marker)
+MARKER_FAR= ( Farfield )
+%
+% ------------------------ SURFACES IDENTIFICATION ----------------------------%
+% Marker(s) of the surface in the surface flow solution file
+MARKER_PLOTTING = ( ACTDISK_IN, ACTDISK_OUT )
+%
+% Marker(s) of the surface where the non-dimensional coefficients are evaluated.
+MARKER_MONITORING = ( ACTDISK_IN, ACTDISK_OUT )
+%
+% Marker(s) of the surface that is going to be analyzed in detail (massflow, average pressure, distortion, etc)
+MARKER_ANALYZE = ( ACTDISK_IN, ACTDISK_OUT )
+%
+% Method to compute the average value in MARKER_ANALYZE (AREA, MASSFLUX).
+MARKER_ANALYZE_AVERAGE = MASSFLUX
+%
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+%
+% CFL number (initial value for the adaptive CFL number)
+CFL_NUMBER= 4.0
+%
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= NO
+%
+% Objective function in gradient evaluation   (DRAG, LIFT, SIDEFORCE, MOMENT_X,
+%                                             MOMENT_Y, MOMENT_Z, EFFICIENCY, BUFFET,
+%                                             EQUIVALENT_AREA, NEARFIELD_PRESSURE,
+%                                             FORCE_X, FORCE_Y, FORCE_Z, THRUST,
+%                                             TORQUE, TOTAL_HEATFLUX,
+%                                             MAXIMUM_HEATFLUX, INVERSE_DESIGN_PRESSURE,
+%                                             INVERSE_DESIGN_HEATFLUX, SURFACE_TOTAL_PRESSURE,
+%                                             SURFACE_MASSFLOW, SURFACE_STATIC_PRESSURE, SURFACE_MACH)
+% For a weighted sum of objectives: separate by commas, add OBJECTIVE_WEIGHT and MARKER_MONITORING in matching order.
+OBJECTIVE_FUNCTION= DRAG
+%
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+% Linear solver or smoother for implicit formulations:
+% BCGSTAB, FGMRES, RESTARTED_FGMRES, CONJUGATE_GRADIENT (self-adjoint problems only), SMOOTHER.
+LINEAR_SOLVER= FGMRES
+%
```

Final request, please remove comments from the config file, except any comments you added to document this specific feature.
We removed these from other testcases because the list of possible options get outdated quickly as we make changes and so we want to have just the config_template as source of truth.
```suggestion
OBJECTIVE_FUNCTION= DRAG
%
% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
LINEAR_SOLVER= FGMRES
%
```